docs/docusaurus.config.js

```diff
@@ -79,6 +79,11 @@ const config = {
             label: 'GitHub',
             position: 'right',
           },
+          {
+            href: 'https://discord.gg/dHJBZw6ewT',
+            label: 'Discord',
+            position: 'right',
```

Comment:
Not sure if it matters...I was thinking to position it "left" - but I'm okay with it on the right (not a hill I will die on)

Reply:
The GitHub link is on the right, so I think this is exactly where it belongs. Next to it.